compute_endpoint/globus_compute_endpoint/endpoint/endpoint_manager.py

```diff
@@ -483,10 +495,10 @@ def cmd_start_endpoint(
             # exit_code as a last-ditch attempt at sharing "what went wrong where" to
             # the parent process.
             exit_code += 1
-            log.debug("Writing credentials")
-            with os.fdopen(write_handle, "w") as cred_pipe:
+            log.debug("Writing credentials and config to stdin")
+            with os.fdopen(write_handle, "w") as write_pipe:
                 # intentional side effect: close handle
-                cred_pipe.write(amqp_creds)
+                write_pipe.write(stdin_data)
```

Comment:
Three hard things in CS.  Naming in this case.  `write_pipe` is redundant, but you're correct, that `cred_pipe` is no longer a fully accurate description either.  Maybe "stdin_pipe"?  We're clearly `.write(...)`ing to it, and the flag `"w"` flag to `.fdopen()` tells us in context that it's writable.